---
layout: page
title: Pictures
permalink: /Picutres/
---


-------------------------------
<div style="float: left; padding-right: 15px">
    <a href="http://beissingerlab.github.io/img/FieldCrew.jpg"><img src="/img/FieldCrew.jpg" title="2016 Summer Crew" width="400" border="0"></a>
</div>

## 2016 Field Crew
Right to Left: Kate, Alex, Tim. We set our alarms and got out to the field bright and early this summer to get science done.

<br>

------------------
<div style="float: right; padding-left: 15px">
    <a href="http://beissingerlab.github.io/img/pastPresFuture.png"><img src="/img/pastPresFuture.png" title="Past Present Future" width="400" border="0"></a>
</div>

## Genomic prediction
Techniques such as quantitative trait locus (QTL) mapping, Genome Wide Association Studies (GWAS), and genomic selection (a.k.a genomic prediction) all focus on relating genotypes and phenotypes is present-day populations. Conversely, selection-mapping and/or scans for selection focus on identifying loci that were historically important during past selection, without necessarily incorporating any phenotypic information. Therefore, results from selection mapping and other evolutionary studies represent independent information that may prove beneficial when incorporated into a genomic-prediction framework. We are working to develop and implement methods to accelerate plant (and animal) improvement by incorporating evidence of **past selection** into prediction models that emphasize **present genotype-phenotype relationships** to develop **better populations in the future**.


<br>

------------------------

<div style="float: left; padding-right: 15px">
    <a href="http://beissingerlab.github.io/img/geneAction.png"><img src="/img/geneAction.png" title="Gene Action" width="400" border="0"></a>
</div>

## Response to selection
Before the development of genomic selection, phenotypic selection was the primary technique implemented for improving plant and animal populations. Because of this, a multitude of selected maize and other crop populations exist that can be studied to gain a better understanding of how selection impacts the genetic makeup of populations. However, in crops most of these resources were selected with the primary goal of immediate plant improvement, and therefore they are not optimized for evolutionary studies. For instance, large population sizes, replication, and rapid LD-decay are three factors that are missing from many existing selected populations. We are beginning a new long-term selection study designed to facilitate down-stream genetic studies. Important details such as the trait we will select are still being determined (as of July 2015), so stay tuned for exciting updates! Down the road, we will use this population, coupled with the existing resources previusly mentioned, to do mapping, investigate gene action, evaluate effect sizes, and generally improve the understanding of how crops such as maize respond to selection. 

For example, the figure above depicts simulations showing how allele frequencies are expected to change across generations under drift (black lines, both panels), selection at a locus that is additive (red lines, left panel), and selection at a locus that displays overdominance (red lines, right panel). By tracking factors such as allele frequency change over the course of a **replicated** selection experiment, we will be able to determine which of these potential gene-actions is at play.

<br>

------------------------

## More to come